src/components/Tabs/Tabs.jsx

```diff
@@ -2,0 +1,27 @@
+export const Tabs = ({ tabs, activeTab, onTabSelected }) => {
+  return (
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={tab.id === activeTab ? 'is-active' : ''}
```

Comment:
Update the condition to use `activeTabId` instead of `activeTab` to maintain consistency with the prop name change.